var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -136,90 +133,119 @@ class Llvm(CMakePackage, CudaPackage, LlvmDetection, CompilerPackage):
     conflicts("+flang", when="@:10")
     conflicts("+flang", when="~clang")
 
-    variant("lldb", default=True, description="Build the LLVM debugger")
+    # lldb
+    variant("lldb", default=False, description="Build the LLVM debugger")
     conflicts("+lldb", when="~clang")
 
-    variant("lld", default=True, description="Build the LLVM linker")
+    # lld
+    variant("lld", default=False, description="Build the LLVM linker")
+
+    # mlir
     variant("mlir", default=False, when="@10:", description="Build with MLIR support")
+
+    # libunwind
     variant(
         "libunwind",
         values=(
             "none",
             conditional("project", when="@:15"),
             conditional("runtime", when="+clang @6:"),
         ),
-        default="runtime",
+        default="none",
         description="Build the LLVM unwinder library"
         "either as a runtime (with just-build Clang) "
         "or as a project (with the compiler in use)",
     )
+
+    # polly
     variant(
         "polly",
-        default=True,
+        default=False,
         description="Build the LLVM polyhedral optimization plugin, only builds for 3.7.0+",
     )
+    conflicts("+polly", when="@:3.6")
+
+    # libcxx
     variant(
         "libcxx",
         values=(
             "none",
             conditional("project", when="@:15"),
             conditional("runtime", when="+clang @6:"),
         ),
-        default="runtime",
+        default="none",
         description="Build the LLVM C++ standard library "
         "either as a runtime (with just-build Clang) "
         "or as a project (with the compiler in use)",
     )
 
-    variant("offload", default=True, when="@19:", description="Build the Offload subproject")
+    # offload
+    variant("offload", default=False, when="@19:", description="Build the Offload subproject")
     conflicts("+offload", when="~clang")
 
-    variant("libomptarget", default=True, description="Build the OpenMP offloading library")
+    # openmp
+    variant(
+        "openmp",
+        values=("none", "project", conditional("runtime", when="+clang @12:")),
+        default="none",
```

Comment:
This default change is also breaking, needs to be "runtime" here.  Curious about this one, why disable it? Building the entire OpenMP runtime is usually a few seconds.